dbt_coves/tasks/dbt/main.py

```diff
@@ -108,11 +109,62 @@ def run_dbt(self, args: list, cwd: str):
         str_args = " ".join([arg if " " not in arg else f"'{arg}'" for arg in args])
         self.run_command(f"dbt {str_args}", cwd=cwd)
 
+        if self.get_env("DATACOVES__UPLOAD_MANIFEST") in ["True", "true", True]:
```

Comment:
I'd rather use a command argument to decide if manifest should be uploaded. Imagine the scenario were you need to run more than one dbt command in the same DAG. You might not need to upload the manifest sometimes.

Reply:
Noel requested to not make this a command argument so that we can enable the manifest upload behind the scenes and with a single config change.

Reply:
ok, @noel's call then